### DISCUSSION TITLE
The Sample Dashboard "Domain Audit" is abnormal

### DISCUSSION THREAD

**Discussion by lyr3051** [Axway-API-Management-Plus/apigateway-openlogging-elk#182] [Q&A] [answered]
Hi 
I already imported the "Domain Audit" dashboard from the release package 4.0.3, and When I try to publish and unpublish the API in API Manager, the Domain Audit dashboard couldn't count the number of API published, API unpublished, etc.

<img width="1369" alt="image" src="https://user-images.githubusercontent.com/43023893/156542690-91d87469-131c-4f6a-858a-778446f857d6.png">

I found the data of API published, unpublished, etc. is from the "apigw-monitoring*" of Index, but I just have only one apigw-monitoring01, and it doesn't update for a long time.
<img width="721" alt="image" src="https://user-images.githubusercontent.com/43023893/156541896-838a1bae-a209-4b6d-98cc-0bc952e08e65.png">

Do I have to configure something to let the dashboard work successfully?

**Comment by cwiechmann** [comment, marked as answer]
This dashboard is currently not working. Please use the KPI-Dashboard instead.

**Reply by lyr3051** [reply]
@cwiechmann OK~
By the way, what is the "subscription" in the KPI-dashboard mean?
How to trigger the count of "subscription" in this dashboard?

**Reply by cwiechmann** [reply]
In API-Manager select an Application and add another API. This is called a subscription and will result in a change in the KPI dashboard.
